src/api/v2/orders.rs

```diff
@@ -192,4 +221,37 @@ mod tests {
     // There shouldn't be any other orders with the given ID.
     assert_eq!(filtered.next(), None);
   }
+
+  #[test(tokio::test)]
+  #[serial]
+  async fn symbol_filter_orders() {
```

Comment:
Would you mind adding a doc comment of what the test does? I try to do that for newly added tests.